src/renderer/App.tsx

```diff
@@ -1,20 +1,16 @@
 import React, { useState, useEffect, useLayoutEffect } from 'react'
-import { i18nContext } from './contexts'
-import ScreenController from './ScreenController'
-import attachKeybindingsListener from './keybindings'
+import moment from 'moment'
```

Comment:
I'm grouping all external and internal dependencies visually to make it clearer where things come from. I could see that in some files that already was the case. This PR is not about this little detail, but as soon as I touched a file I reorganised them.

This thing can be linted (sorted, grouped imports) + autofixed btw, happy to make a PR with the tooling for that.

Reply:
> This thing can be linted (sorted, grouped imports) + autofixed btw, happy to make a PR with the tooling for that.

sure why not, though only if it does not add too many new dependencies ;)